app/MyApp/app/views/GroupForm.js

```diff
@@ -170,10 +170,10 @@ $(function () {
                 success: function () {
                     //create the form
                     var optns = []
-                   /* optns.push({
-                        label: App.languageDict.attributes.Select_An_option,
-                        val: "0000"
-                    })*/
+                    /* optns.push({
```

Comment:
Commented code should not be pushed/committed. Make sure to delete this code and also remove any un-necassary code whenever you are going to push/commit.